src/tailwindcss-stubs/resources/views/auth/passwords/reset.blade.php

```diff
@@ -16,43 +16,43 @@
-                                <p class="text-red text-xs italic mt-4">
+                                <p class="text-red-500 text-xs italic mt-4">
                                     {{ $errors->first('email') }}
                                 </p>
                             @endif
                         </div>
 
                         <div class="flex flex-wrap mb-6">
-                            <label for="password" class="block text-grey-darker text-sm font-bold mb-2">
+                            <label for="password" class="block text-gray-700 text-sm font-bold mb-2">
                                 {{ __('Password') }}:
                             </label>
 
-                            <input id="password" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker leading-tight focus:outline-none focus:shadow-outline{{ $errors->has('password') ? ' border-red' : '' }}" name="password" required>
+                            <input id="password" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline{{ $errors->has('password') ? ' border-red' : '' }}" name="password" required>
 
                             @if ($errors->has('password'))
-                                <p class="text-red text-xs italic mt-4">
+                                <p class="text-red-500 text-xs italic mt-4">
                                     {{ $errors->first('password') }}
                                 </p>
                             @endif
                         </div>
 
                         <div class="flex flex-wrap mb-6">
-                            <label for="password-confirm" class="block text-grey-darker text-sm font-bold mb-2">
+                            <label for="password-confirm" class="block text-gray-700 text-sm font-bold mb-2">
                                 {{ __('Confirm Password') }}:
                             </label>
 
-                            <input id="password-confirm" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-grey-darker leading-tight focus:outline-none focus:shadow-outline" name="password_confirmation" required>
+                            <input id="password-confirm" type="password" class="shadow appearance-none border rounded w-full py-2 px-3 text-gray-700 leading-tight focus:outline-none focus:shadow-outline" name="password_confirmation" required>
                         </div>
 
                         <div class="flex flex-wrap">
-                            <button type="submit" class="bg-blue hover:bg-blue-dark text-white font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
+                            <button type="submit" class="bg-blue-500 hover:bg-blue-700 text-gray-100 font-bold font-bold py-2 px-4 rounded focus:outline-none focus:shadow-outline">
```

Comment:
Seem to have mistakenly doubled up on the `font-bold` class here.

Reply:
Good catch, thanks @imliam